src/README.md

```diff
@@ -28,14 +28,14 @@ an `apb.yaml`, a `Dockerfile`, a `roles` directory, and a `playbooks` directory.
 APB name.
 
 #### Options:
-
+```
 `--organization ORG` specifies an organization name for the APB image on the `Dockerfile`. Defaults to `ansibleplaybookbundle`.
 `--async ASYNC_TOPION` specifies the asynchronous operation of the APB. Defaults to `optional`
```

Comment:
s/TOPION/OPTION

I still think `--async` should be a boolean instead of having the user give a specific value.